CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `transparent` bgcolor
```

Comment:
Move this line to `### Changed` section, something like:
- Transparent mode support for `bgcolor` parameter